app/packs/stylesheets/workEditor.scss

```diff
@@ -74,4 +69,24 @@
   div.embargo-date {
     margin-bottom: 1rem !important;
   }
+
+  .contributor-row.inner-container {
```

Comment:
Is it necessary to have `.inner-container` here? Why can't we just put this under `.contributor-row`?

Reply:
It gets trumped by `.work-editor section .inner-container` in `editor.scss`, which sets a different margin.

Reply:
Please add a comment about this so there's no accidental refactoring to something simpler.

Reply:
Done.